setup_scripts/moab_setup.sh

```diff
@@ -9,16 +9,22 @@ mkdir bld
 MOAB_INSTALL_DIR=$INSTALL_ROOT/MOAB/install
 mkdir $MOAB_INSTALL_DIR
 git clone https://bitbucket.org/fathomteam/moab
```

Comment:
Same comment as above, `git clone` provides a nice `-b` option for checking out a specific branch when cloning. Might be nice to use here.